riscof-plugin/sail_cSim/sail_platform.yaml

```diff
@@ -0,0 +1,4 @@
+nmi:
+  label: nmi_vector
```

Comment:
What are these things? These aren't defined anywhere in the Sail model nor in your PR here. These are just symbols that the old RISC-V ISA tests happen to use, but have no special meaning to the model.

Reply:
These are forward looking; SAIL doesn't currently support interrupts in general- but to be able to test interrupts, that support will be added (in a fairly restricted way). We are working on how that will interface to the tests first, before incorporating it into the Sail csim.

Reply:
So then don't add it here until it's a thing that exists?

Reply:
Why not? The spec defines the concept of a platform defined NMI vector, so the platform YAML includes it.
But: this is part of a larger discussion of why Sail even needs its own YAML if the intent is to configure it to match some other model. Part of that is to make all plugins have the same format, so everything is interchangeable (e.g. we could have spike as a reference and compare the sail csim to it instead of the other way around. I'm not sure how good an excuse that is, so we will discuss that.

Reply:
Defining an NMI vector is fine, but not if the thing you're defining it to doesn't exist, or is a property of the tests your running rather than the model (in which case it should be the test runner that knows what the NMI vector for the test should be).

Reply:
The YAML is there to describe what the vendor has implemented according to the RISC-V spec.
There will be tests for this in the not too distant future, and we don't want to make vendors have to go back and modify their YAML as if they were changing their implementation when those tests are added to the ACT repo- because they're not. Only the tests are changing.